concent_api/core/views.py

```diff
@@ -93,20 +123,39 @@ def send(_request, client_message):
 
         assert force_report_computed_task.task_to_compute.compute_task_def['task_id'] == client_message.cannot_compute_task.task_to_compute.compute_task_def['task_id']
         if client_message.cannot_compute_task.reason == message.CannotComputeTask.REASON.WrongCTD:
-            store_message(
+            client_message.sig = None
+            golem_message, message_timestamp = store_message(
                 client_message.TYPE,
                 force_report_computed_task.task_to_compute.compute_task_def['task_id'],
                 client_message.serialize()
             )
+            store_receive_message_status(
+                golem_message,
+                message_timestamp,
+            )
+            store_message_auth(
+                golem_message,
+                provider_public_key  = force_report_computed_task_from_database.last().auth.provider_public_key,
+                requestor_public_key = client_public_key,
+            )
```

Comment:
Then it should be retrieved using `Message.objects.get()` method and DoesNotExists exception might be handled by using 
```        if not force_report_computed_task_from_database.exists():
            raise Http400("'ForceReportComputedTask' for this task has not been initiated yet. Can't accept your 'RejectReportComputedTask'.")
``` 
this code, but how would you propose to handle MultipleObjectsReturned error ?

Reply:
Changed to use Message.objects.get and proper exceptions handling.

Reply:
Not really. We don't use `get()` because it's not certain that there's exactly one object. There might be none.

> but how would you propose to handle MultipleObjectsReturned error

You should not handle it. It will never happen. But I want this to be stated explicitly and in a way that Python can check and enforce. That's why I want an `assert` and not an exception here. This is just a sanity check that runs only in development. It's like saying ''I know that this instruction can technically return two objects but I assert that a correctly written send() will prevent this from ever happening.".

By writing this explicitly with `assert` you let Python call you out on this if you're wrong. And it helps anyone analyzing the code know that we don't have to worry about this case.

Reply:
:+1: